test/dox/CMakeLists.txt

```diff
@@ -0,0 +1,39 @@
+# Minimum cmake version
```

Comment:
name the folder `test/documentation/` maybe? I think it would be clearer. Also our old seqan2 documentation system was actually called dox so this would be confusing...

Reply:
yeah I first wanted to get this working